README.md

```diff
@@ -3,11 +3,12 @@ GOV.UK Publishing Components ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 =====================
 
-This gem:
+GOV.UK Publishing Components is a Ruby gem to document and distribute components in GOV.UK frontend applications.
```

Comment:
Nitpick: is `GOV.UK frontend` confusing here? I initially read it as `govuk-frontend`.

Similar problem on line 38, we're talking about `GOV.UK frontend` and `GOV.UK Frontend`.

Reply:
"in GOV.UK applications" perhaps? I get the confusion, harder to stay away from it.

Reply:
Yeah, I couldn't think of a good alternative. GOV.UK publishing?

Reply:
GOV.UK **Publishing** as you are the publishing service part of GOV.UK (government online) 😈

Reply:
Maybe?

```suggestion
GOV.UK Publishing Components is a Ruby gem to document and distribute components for GOV.UK frontend applications.
```

Reply:
My only concern with using 'GOV.UK Publishing' is that people may perceive them as only the ones sitting on a `.publishing.service.gov.uk` domain (e.g. Signon, Content Publisher) instead of the entire suite of applications GOV.UK owns (including the ones that are visible on www.gov.uk).

Reply:
I agree with Ian and Alex:

```suggestion
GOV.UK Publishing Components is a Ruby gem to document and distribute components for GOV.UK applications.
```

Reply:
GOV.UK is Government online, it's a problem that GOV.UK Publishing sees it self as 'all of GOV.UK', it should see itself as a service like Verify, Notify, Pay.

GOV.UK Publishing is the programme on the 6th floor that owns all of those suite of applications.

Reply:
I agree with you in principle, but I don't think this readme is necessarily the right place to blaze a trail for that distinction.

There's a trade-off we need to make between potentially confusing external users (who don't understand what we mean when we say "GOV.UK" as opposed to "GOV.UK Publishing"), and confusing internal users (who already use the phrase "GOV.UK publishing applications" to refer to a subset of the applications that use this gem).

Maybe: _GOV.UK Publishing Components is a Ruby gem to document and distribute components for applications maintained by the GOV.UK team at GDS_ would be less controversial?

At least removing "frontend" clears up the confusion with "govuk-frontend" though.